plugins/woocommerce-admin/client/tasks/reminder-bar/reminder-bar.tsx

```diff
@@ -160,7 +160,7 @@ export const TasksReminderBar: React.FC< ReminderBarProps > = ( {
 
 	const tracksProps = {
 		completed: completedTasksCount,
-		is_homescreen: isHomescreen,
+		is_homescreen: !! isHomescreen,
```

Comment:
This seemed to cause a TS issue for me. This enforces it to always be a boolean.